packages/mdc-tabs/README.md

```diff
@@ -0,0 +1,414 @@
+
+  if (dotIndex >= 0) {
+    dynamicTabBar.foundation_.switchToTabAtIndex(dotIndex, false);
+  }
+  updatePanel(dotIndex);
+  updateDot(dotIndex);
+})
+```
+
+### Using the CSS-Only Component
+
+`mdc-tab-bar` ships with css for styling a tab layout according to the Material
+Design spec. To use CSS only tab bars, simply use the available class names.
+Note the available `mdc-tab--active` modifier class. This is used to denote the
+currently active tab.
+
+```html
+<nav class="mdc-tab-bar">
+  <a class="mdc-tab mdc-tab--active" href="#one">Item One</a>
+  <a class="mdc-tab" href="#two">Item Two</a>
+  <a class="mdc-tab" href="#three">Three</a>
+  <span class="mdc-tab-bar__indicator"></span>
+</nav>
+```
+
+### Using the JavaScript Component
+
+`mdc-tab-bar` ships with a Component / Foundation combo for `mdc-tab` (a tab),
+`mdc-tab-bar`(a collection components of type MDC-Tab), and
+
+
+#### Including in code
+
+##### ES2015
+
+```javascript
+import {MDCTab, MDCTabFoundation} from '@material/tabs';
+import {MDCTabBar, MDCTabBarFoundation} from '@material/tabs';
+```
+
+##### CommonJS
+
+```javascript
+const mdcTabs = require('@material/tabs');
+const MDCTab = mdcTabs.MDCTab;
+const MDCTabFoundation = mdcTabs.MDCTabFoundation;
+
+const MDCTabBar = mdcTabs.MDCTabBar;
+const MDCTabBarFoundation = mdcTabs.MDCTabBarFoundation;
+```
+
+##### AMD
+
+```javascript
+require(['path/to/@material/tabs'], mdcTabs => {
+  const MDCTab = mdcTabs.MDCTab;
+  const MDCTabFoundation = mdcTabs.MDCTabFoundation;
+
+  const MDCTabBar = mdcTabs.MDCTabBar;
+  const MDCTabBarFoundation = mdcTabs.MDCTabBarFoundation;
+});
+```
+
+##### Global
+
+```javascript
+const MDCTab = mdc.tabs.MDCTab;
+const MDCTabFoundation = mdc.tabs.MDCTabFoundation;
+
+const MDCTabBar = mdc.tabs.MDCTabBar;
+const MDCTabBarFoundation = mdc.tabs.MDCTabBarFoundation;
+```
+
+#### Automatic Instantiation
+
+If you do not care about retaining the component instance for the tabs, simply
+call `attachTo()` and pass it a DOM element.
+
+```javascript
+mdc.tabs.MDCTabBar.attachTo(document.querySelector('#my-mdc-tab-bar'));
+```
+
+#### Manual Instantiation
+
+Tabs can easily be initialized using their default constructors as well, similar
+to `attachTo`. This process involves a factory to create an instance of MDCTab
+from each tab Element inside of the `mdc-tab-bar` node during the intialization phase
+of `MDCTabBar`, e.g.:
+
+```html
+<nav id="my-mdc-tab-bar" class="mdc-tab-bar">
+  <a class="mdc-tab mdc-tab--active" href="#one">Item One</a>
+  <a class="mdc-tab" href="#two">Item Two</a>
+  <a class="mdc-tab" href="#three">Three</a>
+  <span class="mdc-tab-bar__indicator"></span>
+</nav>
+```
+
+```javascript
+import {MDCTabBar, MDCTabBarFoundation} from '@material/tabs';
+
+const tabBar = new MDCTabBar(document.querySelector('#my-mdc-tab-bar'));
+```
+
+## Tab Section
+
+### Tab component API
+
+#### MDCTab.destroy() => void
```

Comment:
I added a `setter/getter` section to the component API docs to address this. Let me know if there is a more preferred notation. I couldn't really find any examples of documenting instances of public getters AND setters. This seems logical to me